packages/global-cli/package.json

```diff
@@ -12,7 +12,8 @@
     "url": "https://github.com/react-native-community/react-native-cli.git"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "link-package": "yarn link"
```

Comment:
please revert, we don't plan to touch this package

Reply:
If I revert this then the script will break. It expects to find it in all packages 😭

```
> react-native-cli
error Command "link-package" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 1
Command: /home/molant/.nvm/versions/node/v13.6.0/bin/node
Arguments: /usr/share/yarn/lib/cli.js link-package
Directory: /home/molant/projects/react-native/cli/packages/global-cli
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspaces for documentation about this command.   error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

There [was a proposal](https://github.com/yarnpkg/yarn/issues/6894) about adding `--if-present` flag like in npm but it is stale right now.

If not touching `global-cli` is a priority then I'll probably have to work on a script to make it work cross-platform.

Reply:
Nah, I think we can configure lerna to ignore it when publishing. Will check tomorrow

Reply:
This is a known issue with running `yarn workspace run`. You should be able to do `exit 0;` instead of `yarn link` to just indicate no action needed. That's what I've done before.

Reply:
@grabbou but we still need to update that package, right? If we are OK with updating `global-ci` then I'll change it to `exit 0`

Reply:
@thymikee I've added `global-cli` to the ignore section in `lerna.json` in a separate commit. The output of running `lerna changed` is:

```
$ yarn lerna changed
yarn run v1.21.1
$ /home/molant/projects/react-native/cli/node_modules/.bin/lerna changed
lerna notice cli v3.18.4
lerna info Looking for changed packages since v3.0.4
lerna info ignoring diff in paths matching [ 'global-cli' ]
@react-native-community/cli-types
@react-native-community/cli
@react-native-community/cli-platform-android
@react-native-community/cli-platform-ios
lerna success found 4 packages ready to publish
Done in 0.52s.
```

Reply:
Lovely! Will check as soon as I can